pkg/systemd/host.css

```diff
@@ -0,0 +1,31 @@
+.systime-inline {
```

Comment:
This is for the list of ntp servers, #ntp-servers.  I copied it from the docker port config markup, which is very similar.